runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -353,7 +382,6 @@ NNFW_STATUS nnfw_session::load_model_from_nnpackage(const char *package_dir)
       return NNFW_STATUS_ERROR;
     }
     _model->primary_subgraph()->bindKernelBuilder(_kernel_registry->getBuilder());
-    _coptions = onert::compiler::CompilerOptions::fromGlobalConfig();
```

Comment:
Just note.

- `_coptions` will be a collection (i.e. `std::vector<CompilerOptions>`) for the possibility of multiple model of nnpkg. 
- Two internal `load...` APIs are used for one model test only. Thus, moving this into `nnfw_create` is okay.
- Besides, `load_model_from_nnpackage` may have multiple models, which requires multiple `CompilerOptions`, and it is determined after load. It will be here again.